src/stdlib/File.ts

```diff
@@ -1,16 +1,15 @@
-import { Callable, ValueKind, BrsString, BrsBoolean } from "../brsTypes";
+import { Callable, ValueKind, BrsString, BrsBoolean, BrsArray } from "../brsTypes";
 import { Interpreter } from "../interpreter";
-import { Volume } from "memfs/lib/volume";
 import { URL } from "url";
-
+import MemoryFileSystem from "memory-fs";
 
 /*
  * Returns a memfs volume based on the brs path uri.  For example, passing in
  * "tmp:///test.txt" will return the memfs temporary volume on the interpreter. 
  * 
  * Returns invalid in no appopriate volume is found for the path
  */
-export function getVolumeByPath(interpreter: Interpreter, path: string): Volume | null {
+export function getVolumeByPath(interpreter: Interpreter, path: string): MemoryFileSystem | null {
```

Comment:
Obviously it wasn't changed here but I don't think we need to export `getVolumeByPath` – seems like it's only used within this file and for testing, right?  I ask only because it gets exported with the rest of `StdLib` even though it's not a `Callable`.  It might be worth moving this (and `getMemFsPath`) into a separate file so they don't get exported with the rest of the standard library, but that can happen later.

Reply:
Ah, when I started writing this the functions weren't auto-imported so I thought it was fine for testing.  I see the auto-import already filters out non-Callables.  What's the concern of leaving this here for testing?  Is there a better way to expose this so it can still be testing?

Reply:
Oh right!  I forgot how the timing worked out there 😅 

I'm mostly concerned with exporting what is essentially a private method – I'm not sure you'd want anyone using this outside of `File.ts`, so exporting it makes the tacit statement that it's okay to do that.  Probably the only way to expose it for testing without exporting it out of `StdLib` is to pull these two functions into a separate file (`FileUtils.ts` or something) and test them by importing that file directly.  They could still be used here by importing `FileUtils`, but they wouldn't get exported from `StdLib` because `stdlib/index.ts` wouldn't have an `export * from "./FileUtils";` 😄

Reply:
I'm not sure I see the different if stdlib is filtering out the non-Callables.  I did give it a brief attempt, but ran into some import issues due to my inexperience.  We can talk more about this offline.